tests/Tests/200__monitor_and_manage/200__metrics/206__pvc_usage_90.robot

```diff
@@ -0,0 +1,58 @@
+*** Settings ***
+Force Tags       Smoke  Sanity
+Resource         ../../../Resources/ODS.robot
+Resource         ../../../Resources/Common.robot
+Resource         ../../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource         ../../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library          DebugLibrary
+Library          SeleniumLibrary
+Library          JupyterLibrary
+
+*** Variables ***
+${python_code}=  SEPARATOR=\n
+...  import os
+...  with open('large_file_over90.tmp', 'wb') as fout:
+...  \t fout.write(os.urandom(1841976115))
+${alert_name}=  RHODS-PVC-Usage
+${alert_description}=  User notebook pvc usage above 90%
+
+*** Test Cases ***
+Set Up JupyterLab
+  Set Library Search Order  SeleniumLibrary
+
+  Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+  Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+  Wait for RHODS Dashboard to Load
+  Launch JupyterHub From RHODS Dashboard Dropdown
+
+  Login To Jupyterhub  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+  ${authorization_required} =  Is Service Account Authorization Required
+  Run Keyword If  ${authorization_required}  Authorize jupyterhub service account
+  Wait Until Page Contains Element  xpath://span[@id='jupyterhub-logo']
+  Spawn Notebook With Arguments  image=s2i-minimal-notebook
+  Wait for JupyterLab Splash Screen  timeout=30
+
+  Maybe Select Kernel
+  ${is_launcher_selected} =  Run Keyword And Return Status  JupyterLab Launcher Tab Is Selected
+  Run Keyword If  not ${is_launcher_selected}  Open JupyterLab Launcher
+  Launch a new JupyterLab Document
+  Close Other JupyterLab Tabs
+  Sleep  5
+
+Trigger PVC 90
+  Add and Run JupyterLab Code Cell  ${python_code}
+  Capture Page Screenshot
+
+Check Prometheus
+  Sleep  300
+  Prometheus.Check Alert Triggered  ${RHODS_PROMETHEUS_URL}  ${RHODS_PROMETHEUS_TOKEN}  alert  ${alert_name}  ${alert_description}
+
+Stop JupyterLab
```

Comment:
You don't need this when using the common Suite Teardown

`Test Teardown    Common.End Web Test`

Reply:
Yeah, that should work with 206, unfortunately, with 207, as the notebook is full, it's impossible to open a new page, so it need to be cleaned up in the same cell. But it's a great idea thanks!

Reply:
In your Test Teardown (instead of Suite Teardown) you can run this notebook to clean up the space

https://github.com/redhat-rhods-qe/ods-ci-notebooks-main/blob/main/notebooks/200__monitor_and_manage/203__alerts/notebook-pvc-usage/fill-notebook-pvc-delete-testfiles.ipynb